wallet/Cargo.toml

```diff
@@ -13,11 +13,17 @@ crate-type = ["cdylib"]
 path = "src/lib.rs"
 
 [dependencies]
+base64 = "0.13.0"
 ic-types = "0.1.2"
-ic-cdk = "0.2.3"
-ic-cdk-macros = "0.2.3"
+ic-cdk = { git = "https://github.com/dfinity/cdk-rs", branch = "main"}
+ic-cdk-macros = { git = "https://github.com/dfinity/cdk-rs", branch = "main"}
+ic-certified-map = { git = "https://github.com/dfinity/cdk-rs", branch = "main"}
```

Comment:
can we add a `rev =` here to pin it to specific revision

Reply:
done